recipes/catch2/2.x.x/conanfile.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class ConanRecipe(ConanFile):
+    name = "catch2"
+    description = "A modern, C++-native, header-only, framework for unit-tests, TDD and BDD"
+    topics = ("conan", "catch2", "header-only", "unit-test", "tdd", "bdd")
+    url = "https://github.com/catchorg/Catch2"
```

Comment:
**url** means where the recipe is from, in this case `https://github.com/conan-io/conan-center-index`